<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubtitleSetting</class>
 <widget class="QWidget" name="SubtitleSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleSubtitle">
     <property name="text">
      <string>字幕</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QComboBox" name="subtitleComboBox"/>
     </item>
     <item>
      <widget class="QToolButton" name="loadSubtitleButton">
       <property name="text">
        <string>载入</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="titleSynchronize">
     <property name="text">
      <string>同步</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QToolButton" name="subtitleDefualtsynchronizeTimeButton">
       <property name="text">
        <string>默认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="subtitleSynchronizeTimeBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="titlePosition">
     <property name="text">
      <string>位置</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>底部</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="subtitlePositionBar">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>顶部</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="titleFont">
     <property name="text">
      <string>字体</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QFontComboBox" name="subtitleFontComboBox"/>
     </item>
     <item>
      <widget class="QSpinBox" name="subtitleSizeBox"/>
     </item>
     <item>
      <widget class="QToolButton" name="subtitleBoldButton">
       <property name="text">
        <string>B</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="subtitleItalicsButton">
       <property name="text">
        <string>I</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="subtitleUnderlineButton">
       <property name="text">
        <string>U</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QPushButton" name="subtitleColor">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px dashed white;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>透明度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="subtitleTransparencyBar">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="subtitleTransparencyLabel">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>0%</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QCheckBox" name="subtitleStrokeCheckBox">
       <property name="text">
        <string>字体描边</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="subtitleStrokeColor">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px dashed white;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>透明度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="subtitleStrokeTransparencyBar">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="subtitleStrokeTransparencyLabel">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>0%</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
